### ISSUE TITLE
Blueprint Option: Spellcheck

### ISSUE THREAD

**Issue by Werbschaft** [bogdancondorachi/kirby-table-field#5]
It would be great if the field also supported the "spellcheck: false" option. Just a feature request

**Comment by gbdesign2023** [comment]
> It would be great if the field also supported the "spellcheck: false" option. Just a feature request

‚spellcheck‘ is configurable for ‚text’ and ‚textarea‘ fields. But for tables it is input fields. I don't know of a Kirby setting "spellcheck: false" for input fields...?

**Comment by bogdancondorachi** [comment]
I would disable it by default. In a table case, I don't see making an impact. I could also make it configurable but I just don't see its point here.

**Comment by Werbschaft** [comment]
@gbdesign2023 most off the fields have this option, see here in bottom of the table: https://getkirby.com/docs/reference/panel/fields/text

@bogdancondorachi that would also make sense of course! I personally disable it at nearly each field

**Comment by gbdesign2023** [comment]
@Werbschaft 
Correct. For text and textarea. But the table plugin uses input fields, not text fields. So I can't find a Kirby function to disable spellcheck by option in the blueprint.

I also agree that spellcheck has no advantage in a table.

**Comment by Werbschaft** [comment]
@gbdesign2023 yes, but of course, the text field is a simple input field right? But never mind, it would make sense to add `spellcheck="false"` to all input elements in the field

**Comment by gbdesign2023** [comment]
@Werbschaft That's right! But that would be more of a core function for Kirby to adapt this option to other fields.

**Comment by bogdancondorachi** [comment]
As the new release would replace the `text-input` with `writer-input`, I would go for a config like this:

```vue
<k-writer-input
  v-model="row[columnIndex]"
  :inline="true"
  :nodes="false"
  :spellcheck="false"
  @input="update()"
/>
```

**Comment by Werbschaft** [comment]
Would be amazing to see the writer field in the table

**Comment by bogdancondorachi** [comment]
@Werbschaft This should be fixed in the latest release, you can also find the new writer-input.

**Comment by Werbschaft** [comment]
Amazing work! You're an real artist with vue.js
